canonical-web-and-design/ubuntu.com
# Make the release cycle ids unique

## Done
- Make the aria-controls id's unique to the release-cycle section
- Removed the id on the logo to stop this happening again the future 

## QA
- Open the demo and go to /about/release-cycle#ubuntu-kernel-release-cycle
- Click the "Ubuntu kernel" tab and see the site logo does not disappear

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/11206
